src/main/java/lv/ctco/javaschool/goal/boundary/GoalApi.java

```diff
@@ -76,7 +77,8 @@ public GoalDto findGoalDtoByGoalId(@PathParam("id") Long goalId) {
     public void saveGoal(GoalFormDto goalDto) {
         User user = userStore.getCurrentUser();
         Goal goal = new Goal();
-        if (goalDto.getGoalMessage() != null && goalDto.getDeadline() != null) {
+        if (goalDto.getGoalMessage() != null && goalDto.getDeadline() != null
```

Comment:
I assume that in case when we are checking goalMessage for null also to check it for emptiness.
``` StringUtils.isNotBlank(goalDto.getGoalMessage()) ``` this method checks for null and that string is not empty

Reply:
This method requires apache library, so empty check is done with ```!goalDto.getGoalMessage().trim().isEmpty```